index.html

```diff
@@ -4,7 +4,64 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
     <title>setup-mb-skeleton</title>
+    <link rel="stylesheet" href="style.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1>My logo</h1>
+      <img src="images/Menu.svg" alt="hamburger" />
+    </header>
+    <main class="wrapper">
+      <div class="banner">
+        <img src="images/Ellipse5.svg" alt="" />
+        <img src="images/Ellipse15.svg" alt="" class="top-circ icone-btn" />
+        <img src="images/Rectangle7.svg" alt="" />
+        <img src="images/Ellipse11.svg" alt="" class="aside-elip icone-btn" />
+      </div>
+      <section class="container">
+        <p class="primary-text">I'm Christian<br />Glad to see<br />you!</p>
+        <img src="images/Ellipse3.svg" alt="" class="big-elips icone-btn" />
```

Comment:
- [ ] This is a header of the page, therefore, please use `h1` tag instead of `<p>`